### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by bovirus**

### CODE REVIEW

File: recovery/translation_it.ts [hunk 1]
```diff
@@ -1788,8 +1788,8 @@ Tuttavia, a differenza di vncinstall, qualsiasi tastiera e mouse collegati local
         <location filename="optionsdialog.ui" line="486"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables an SSH connection to the rescueshell in PINN. It cannot be used to control PINN, but having a shell is very useful to perform remote maintenance on any of the SD card partitions.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
         <oldsource>Enables an SSH connection to the rescueshell in PINN. It cannot be used to control PINN, but having a shell is very useful to perform remote maintenance on any of the SD card partitions</oldsource>
-        <translation>Abilita una connessione SSH al rescueshell in PINN. 
-Non può essere usata per controllare PINN, ma avere una shell è molto utile per eseguire la manutenzione remota in una qualsiasi delle partizioni della scheda SD.</translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Abilita una connessione SSH al rescueshell in PINN.
```
Comment (lurch) [inline recovery/translation_it.ts:1791]: Ummmm, WTF? :slightly_frowning_face:

**Comment by bovirus** [comment]
@lurch 

The new translation about these string reflect the new layout of master English language that include special chars before and after.

What's the meaning of "WTF"?

**Comment by lurch** [comment]
Sorry, I should have been clearer. I'm assuming the inclusion of `&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;`  is unintentional?

EDIT: Oops, I hadn't spotted that these are also in the original English! Sorry @bovirus 
Is that an error @procount ?

**Comment by procount** [comment]
I wish I never started adding tooltips to the options dialog and simply cutting-and-pasting straight from the documentation page! 
Who'd have thought it would have caused so many commits, PRs and issues!

**Comment by procount** [comment]
Yes, my cut and paste error as the HTML code does not appear is any(?) of the other tooltips.
Don't worry @bovirus, I will correct it.

**Comment by lurch** [comment]
One method that can cut down on the number of control-characters / hidden-markup that get copied when doing copy'n'paste from a web-page, is to copy from a web-page, paste into a plain-text editor (e.g. gedit) and then copy from the plain-text editor into the final destination.

**Comment by bovirus** [comment]
@procount 

I found also some errors on other language on the use of CR/LF that didn't match the original master English language.

I beleieve that the translation have to match the original English string.

Then if the error was in the original strings we have to fix on the original then re-align translation.

**Comment by procount** [comment]
I am quite happy to modify the original strings from the source code if you find any errors.
Feel free to update any of the other languages and I will merge them in. :wink:

**Comment by bovirus** [comment]
@procount 

My proposal is to align in fisrt step translations to master English language as end punctation etc.

Then if we decide together that we need to change something we will change in master and tarnslation in same way.

Then please apply my latest PR about Italian language.

Then apply also the new PR about other languages.

https://github.com/procount/pinn/pull/286

Then you will have master English aligned to other languages.

Then we will think as modify the source and then reflect it on the translation.

**Comment by procount** [comment]
I have just pushed your last IT translations, plus the previous DE PR.

**Comment by bovirus** [comment]
@procount 

Please close this PR.

**Pull request closed without merging**